pysat/utils/_core.py

```diff
@@ -198,6 +199,11 @@ def load_netcdf4(fnames=None, strict_meta=False, file_format='NETCDF4',
         (default='NETCDF4')
     epoch_name : str
         Data key for time variable (default='Epoch')
+    epoch_unit : str
+        Units of epoch data to convert to datetime (default='ms')
+    epoch_origin : str or timestamp-convertable
+        Origin of epoch calculation, following convention for
+        `pandas.to_datetime`.  (default='unix')
```

Comment:
Why not make the default a timestamp with the unix epoch instead of a string?  This will simplify the logic and provide a clear input example.